app/jupyterlite.schema.v0.json

```diff
@@ -101,12 +101,6 @@
           "default": "./lab/build/themes",
           "$ref": "#/definitions/non-trailing-slash-uri"
         },
-        "pyodideUrl": {
```

Comment:
Nice to see that one removed from the top level :+1:

Reply:
yeah, it was a bit of a haul to get there, but seems clearer. This is really the only thing that would constitute a `v1`, and would be _very_ easy to forward/back migrate... but anybody that has been customizing this already will be impacted by other changes in this PR even more.

once extracted is a dedicated extension, we can figure out some way to move the schema someplace over there that will make it to the built environment such that the `jupyter lite check` step can verify against sub-schema.